### PR TITLE
fix bug with error in Flowable

### PR DESCRIPTION
**Pull request by Toni111111**
### What does this PR do?
This PR fix bug  with errors such as UndeliverableException and etc.

### Where should the reviewer start?
File JsonRpc2_0Rx

### Why is it needed?
Because it fix crashs and erros.

### CODE REVIEW

**Comment by Toni111111** [comment]
why did  The Travis CI build fail ?

**Comment by Toni111111** [comment]
@conor10 @iikirilov why ?

**Comment by iikirilov** [comment]
@Toni111111 there is code style violations. To fix them run `./gradlew spotlessApply` - you can see this on line 277 in the logs of your build.

**Comment by iikirilov** [comment]
Could you add a test that would fail if your code is not in there to help me understand better what the crux of the issue is please.

File: core/src/main/java/org/web3j/protocol/rx/JsonRpc2_0Rx.java [hunk 1]
```diff
@@ -53,19 +53,31 @@ public JsonRpc2_0Rx(Web3j web3j, ScheduledExecutorService scheduledExecutorServi
     public Flowable<String> ethBlockHashFlowable(long pollingInterval) {
         return Flowable.create(
                 subscriber -> {
-                    BlockFilter blockFilter = new BlockFilter(web3j, subscriber::onNext);
-                    run(blockFilter, subscriber, pollingInterval);
+                    try {
+                        if(!subscriber.isCancelled()){
```
Comment (iikirilov) [inline core/src/main/java/org/web3j/protocol/rx/JsonRpc2_0Rx.java:57]: it feels weird that you are checking a property on the subscriber here and then passing the subscriber into the `run(...)` method. Could you not perform this check inside the `run(...)` method to save the duplication of code?

**Comment by AlexandrouR** [comment]
I did a restart on the CI and everything looks good.

**Comment by Toni111111** [comment]
@conor10  i am waiting review :)

**Pull request closed without merging**